virtme/commands/run.py

```diff
@@ -968,6 +969,46 @@ def cleanup_console_script():
     qemuargs.extend(["-device", "vhost-vsock-pci,guest-cid=%d" % args.port])
 
 
+def ssh_client(args):
+    user = f"{args.user}@" if args.user else ""
+
+    if args.remote_cmd is not None:
+        exec_escaped = args.remote_cmd.replace('"', '\\"')
+        remote_cmd = f'bash -c "{exec_escaped}"'
+    else:
+        remote_cmd = ""
+
+    cmd = ["ssh", "-p", str(args.port), f"{user}localhost"] + (
+        [remote_cmd] if remote_cmd else []
+    )
+    if args.dry_run:
+        print(" ".join(cmd))
+    else:
+        os.execvp("ssh", cmd)
+
+
+def ssh_server(args, arch, qemuargs, kernelargs):
+    # Check if we need to generate the ssh host keys for the guest.
+    ssh_key_dir = f"{CACHE_DIR}/.ssh"
```

Comment:
(maybe no need to add the `.` in `.ssh`?)

Reply:
If the cache dir was only used by ssh, yes, but considering that it's a generic cache dir (use to cache the mainline precompiled kernels) I think it's good to have a separate dot dir for ssh.

Reply:
OK, I thought there was no risk of conflicts with the kernel versions, so it was fine to call it `ssh` and keep it "visible". But fine for me to keep it hidden.